test/exandra/connection_test.exs

```diff
@@ -210,6 +210,21 @@ defmodule Exandra.ConnectionTest do
     end
   end
 
+  test "all with prefix" do
+    query =
+      Schema |> select([r], r.x) |> Queryable.to_query() |> Map.put(:prefix, "prefix") |> plan()
+
+    assert all(query) == ~s{SELECT x FROM prefix.schema}
+
+    query =
+      Schema |> from(prefix: "first") |> select([r], r.x) |> Map.put(:prefix, "prefix") |> plan()
+
+    assert all(query) == ~s{SELECT x FROM first.schema}
```

Comment:
What's the use case of having a test for setting a prefix and then overriding it?

Reply:
This tests that the prefix from `from` is used even if another prefix is set in the query. I took some of these tests from the `ecto_sql` [adapter tests](https://github.com/elixir-ecto/ecto_sql/blob/b1212914a7cb695ce842184080b97348f85c2772/test/ecto/adapters/postgres_test.exs#L1097).